src/components/battles/animations/shared/Fainting.ts

```diff
@@ -49,6 +50,11 @@ export class Fainting<TGameStartr extends FullScreenPokemon> extends Component<T
                     partyIsWipedText.push(
                         [playerName, " is out of useable Pokemon!"],
                         [playerName, " blacked out!"]);
+                } else if (teamName === "opponent") {
+                    partyIsWipedText.push(
```

Comment:
This only works for one Pokemon right now. You'll need to change it to also show the menu when multiple Pokemon gain experience.